### PR TITLE
Add plotly package

### PR DESCRIPTION
**Pull request by afbarbaro**
### Description
Adds `plotly` to the provided packages. This package offers several additional types of charts and visualizations and complements `bokeh`, which is already included in `pyodide`. Makes it easier to start producing charts without having to install the package in code (because it can be loaded directly using `loadPackagesFromImports`).

### Checklists

- [ ] Add a [CHANGELOG](https://github.com/pyodide/pyodide/blob/main/docs/project/changelog.md) entry
- [x] Add / update tests
- [ ] Add new / update outdated documentation

### CODE REVIEW

File: packages/plotly/test_plotly.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from pyodide_test_runner import run_in_pyodide
+
+
+@run_in_pyodide(packages=["plotly"])
```
Comment (hoodmane) [inline packages/plotly/test_plotly.py:4]: you will probably need:
```suggestion
@pytest.mark.xfail_browsers(node="plotting doesn't work in node")
@run_in_pyodide(packages=["plotly"])
```

**Comment by ryanking13** [comment]
> Makes it easier to start producing charts without having to install the package in code (because it can be loaded directly using `loadPackagesFromImports`).

If the package is installable through `micropip`, we prefer not to add it to the repository if possible. We recently added [`micropip.freeze()`](https://pyodide.org/en/latest/usage/api/micropip-api.html#module-micropip) API which, I think, is suitable for your use case. Could you try that instead?

**Comment by afbarbaro** [comment]
> > Makes it easier to start producing charts without having to install the package in code (because it can be loaded directly using `loadPackagesFromImports`).
> 
> If the package is installable through `micropip`, we prefer not to add it to the repository unless it requires some patch. We recently added [`micropip.freeze()`](https://pyodide.org/en/latest/usage/api/micropip-api.html#module-micropip) API which, I think, is suitable for your use case. Could you try that instead?

Thank you! `micropip.freeze()` looks helpful. I will try it out.

**Comment by afbarbaro** [comment]
> > > Makes it easier to start producing charts without having to install the package in code (because it can be loaded directly using `loadPackagesFromImports`).
> > 
> > 
> > If the package is installable through `micropip`, we prefer not to add it to the repository unless it requires some patch. We recently added [`micropip.freeze()`](https://pyodide.org/en/latest/usage/api/micropip-api.html#module-micropip) API which, I think, is suitable for your use case. Could you try that instead?
> 
> Thank you! `micropip.freeze()` looks helpful. I will try it out.

This worked well for my usecase, so I'm closing out this PR. Thank you @hoodmane !

**Pull request closed without merging**